publication-commons/src/main/java/no/unit/nva/publication/exception/InvalidPublicationException.java

```diff
@@ -9,6 +9,10 @@
     public static final String ERROR_MESSAGE_TEMPLATE =
         "The Publication cannot be published because the following fields are not populated: ";
 
+    public InvalidPublicationException(String message) {
```

Comment:
Generic message is easier to maintain when fields are going to change name or location